hafas_fetcher.py

```diff
@@ -25,12 +27,17 @@ def fetch_and_parse(self, stop_id):
         departures = sorted(departures)
         for n, dep in enumerate(departures):
             for follow in islice(departures, n + 1, None):
-                if dep.symbol == follow.symbol and (
-                    (dep.platform != "" and dep.platform == follow.platform)
-                    or dep.destination == follow.destination
-                ):
-                    dep.follow = follow
-                    break
+                if SYMBOL_IS_GROUP[CONFIG["api_provider"]]:
+                    if dep.symbol == follow.symbol and (
+                            (dep.platform != "" and dep.platform == follow.platform)
```

Comment:
Indentation seems wrong here

Reply:
This is what clicking "reformat code" in PyCharm outputs